sphinx_search/static/js/rtd_sphinx_search.js

```diff
@@ -271,12 +271,16 @@ const generateSingleResult = (resultData, projectName, id) => {
     let h2_element = createDomNode("h2", {class: "search__result__title"});
     h2_element.innerHTML = page_title;
 
-    // If the result is not from the same project,
-    // then it's from a subproject.
-    if (projectName !== resultData.project) {
+    // Results can belong to different projects.
+    // If the result isn't from the current project, add a note about it.
+    const project_slug = resultData.project.slug
+    if (projectName !== project_slug) {
         let subtitle = createDomNode("small", {class: "rtd_ui_search_subtitle"});
-        subtitle.innerText = `(from project ${resultData.project})`;
+        subtitle.innerText = ` (from project ${project_slug})`;
```

Comment:
```suggestion
        subtitle.innerText = ` (result from <em>${resultData.project.name}</em>)`;
```

I think we want the name, not the slug?

Reply:
We don't expose the name in the API (we don't index the name project name in ES).

Reply:
I think this is fine for now, but we should probably consider saving the name in ES as well. In particular for .com where the slugs are a little weird because the org's slug is prefixed.